src/main/java/de/dhbw/mh/rinne/AstBuilder.java

```diff
@@ -61,6 +62,15 @@ public AstNode visitVariableReference(RinneParser.VariableReferenceContext ctx)
     // Team 1
 
     // Team 2
+    @Override
+    public AstNode visitFunctionCall(RinneParser.FunctionCallContext ctx) {
+        CodeLocation codeLoc = getCodeLocation(ctx);
+        String func = ctx.funcCall().getText();
+        List<AstExpressionNode> args = new ArrayList<AstExpressionNode>();
+        ctx.funcCall().actualParameters.forEach(e -> args.add((AstExpressionNode) visit(e)));
```

Comment:
This code makes sense if you're handling an `expression: funcCall` rule. However, since you're now implementing the `funcCall` rule directly, you can simplify this line by removing the unnecessary `.funcCall()` call:

```suggestion
        ctx.actualParameters.forEach(e -> args.add((AstExpressionNode) visit(e)));
```

This accesses the parameters directly from the current context.